### ISSUE TITLE
1D Convolutional Layer

### ISSUE THREAD

**Issue by lars-reimann** [Safe-DS/Library#853]
### Is your feature request related to a problem?

We only have 2D convolutions so far. 1D convolutions can also be useful, e.g. for time series.

### Desired solution

* Add a new class `safeds.ml.nn.layers.Convolutional1DLayer`.
* Superclass: `Layer`
* Constructor parameters: 
  * `output_channel: int`
  * `kernel_size: int`
  * **keyword-only** `stride: int = 1`
  * **keyword-only** `padding: int = 0`
  * Call `_check_bounds` to ensure that:
  * `output_channel` >= 1
  * `kernel_size` >= 1
  * `stride` >= 1
  * `padding` >= 0
* Implementation:
  * Use PyTorch's [`Conv1D`](https://pytorch.org/docs/stable/generated/torch.nn.Conv1d.html) internally
